woodwork/tests/accessor/test_column_accessor.py

```diff
@@ -380,40 +382,43 @@ def test_remove_semantic_tags_raises_error_with_invalid_tag(sample_series):
 
 
 def test_series_methods_on_accessor(sample_series):
-    xfail_dask_and_koalas(sample_series)
+    xfail_koalas(sample_series)
     series = sample_series.astype('category')
     series.ww.init()
 
     copied_series = series.ww.copy()
     assert copied_series is not series
     assert copied_series.ww._schema == series.ww._schema
-    pd.testing.assert_series_equal(series, copied_series)
+    pd.testing.assert_series_equal(to_pandas(series), to_pandas(copied_series))
 
 
 def test_series_methods_on_accessor_without_standard_tags(sample_series):
-    xfail_dask_and_koalas(sample_series)
+    xfail_koalas(sample_series)
     series = sample_series.astype('category')
     series.ww.init(use_standard_tags=False)
 
     copied_series = series.ww.copy()
     assert copied_series is not series
     assert copied_series.ww._schema == series.ww._schema
-    pd.testing.assert_series_equal(series, copied_series)
+    pd.testing.assert_series_equal(to_pandas(series), to_pandas(copied_series))
 
 
 def test_series_methods_on_accessor_returning_series_valid_schema(sample_series):
-    xfail_dask_and_koalas(sample_series)
+    xfail_koalas(sample_series)
     series = sample_series.astype('category')
     series.ww.init()
 
-    sorted_series = series.ww.sort_values()
-    assert sorted_series.ww._schema == series.ww._schema
-    assert sorted_series.ww._schema is not series.ww._schema
-    pd.testing.assert_series_equal(sorted_series, series.sort_values())
+    replace_series = series.ww.replace('a', 'd')
+    assert replace_series.ww._schema == series.ww._schema
+    assert replace_series.ww._schema is not series.ww._schema
+    pd.testing.assert_series_equal(to_pandas(replace_series), to_pandas(series.replace('a', 'd')))
 
 
 def test_series_methods_on_accessor_inplace(sample_series):
-    xfail_dask_and_koalas(sample_series)
+    # TODO: Try to find a supported inplace method for Dask, if one exists
+    if dd and isinstance(sample_series, dd.Series):
+        pytest.xfail('Dask does not support pop.')
```

Comment:
Dask generally does not support inplace operations - not sure if there is something else we can use here instead, or if we just leave it like this.

Reply:
that's interesting... we were able to have it in DataTables I guess since we were actually creating a new DataFrame object and dropping the relevant column.

But I'm fine leaving it like this if Dask itself doesn't support inplace operations

Reply:
Yeah, I think we got away with it in DataTables since we weren't actually using a `pop` call on the underlying dataframe, and as such we weren't attempting any inplace modifications.